dbt/include/bigquery/macros/relations/drop.sql

```diff
@@ -3,3 +3,12 @@
     drop {{ relation.type }} if exists {{ relation }}
   {%- endcall %}
 {% endmacro %}
+
+
+{% macro bigquery__drop_relation_sql(relation) -%}
```

Comment:
Was this written specifically for this work? Or does something else call `bigquery__drop_relation_sql` or `drop_relation_sql`? If it's the former, then we should rework the dependent macros to use the new `dbt-core` versions. Ideally we don't have this file at all, but I think we implemented a generic `drop_relation` macro for some reason (maybe to remove the cascade keyword?).